main.rb

```diff
@@ -0,0 +1,23 @@
+def bubble_sort(arr)
+  number_ofloop = arr.length - 1
+  (0..number_ofloop).each do |_i|
```

Comment:
- [ ] **OPTIONAL** It is a convention in Ruby to use the only `_` for unused parameters, or you could skip the parameters as well and leave it like: `(0..number_ofloop).each do`